homeassistant/components/zwave_js/discovery.py

```diff
@@ -115,6 +115,20 @@ class ZWaveDiscoverySchema:
         product_type={0x0038},
         primary_value=SWITCH_MULTILEVEL_CURRENT_VALUE_SCHEMA,
     ),
+    # Inovelli LZW26 light / fan controller combo using switch multilevel CC
```

Comment:
```suggestion
    # Inovelli LZW36 light / fan controller combo using switch multilevel CC
```
Whoops, I had a typo